packages/storybook/stories/components/form-field/form-field.stories.ts

```diff
@@ -225,19 +225,348 @@ export const Tooltip: StoryObj = {
   `,
 };
 
+export const Checkbox: StoryObj = {
+  tags: ['isHidden'],
+  render: () => html`
+<ods-form-field>
+  <ods-text id="form-field__checkbox-label" slot="label" preset="label">
```

Comment:
FYI no need for the BEM notation on id, this is just for classes
`form-field-checkbox-label` would have been enough